client/editor.cpp

```diff
@@ -332,7 +332,14 @@ enum editor_tool_mode editor_tool_get_mode(enum editor_tool_type ett)
 /**
    Returns TRUE if the *client* is in edit mode.
  */
-bool editor_is_active() { return can_conn_edit(&client.conn); }
+bool editor_is_active()
+{
+  if (editor != nullptr && can_conn_edit(&client.conn)) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
  return editor != nullptr && can_conn_edit(&client.conn);
```

Reply:
nice! this is why you are the code oracle :)